docs/adoc/technicalGuideJS/ScoutJS.adoc

```diff
@@ -1288,10 +1288,66 @@ Output:
 
   <scout-version data-value="16.1.0.002"></scout-version>
 
+== TypeScript Support
+
+Eclipse Scout is written in TypeScript and therefore comes with a fully typed TypeScript API.
+
+This gives you many benefits:
+
+* *More efficient development* +
+Types allows your IDE to better assist with code completion during development and only suggests methods and properties for an object that actually exist. +
```

Comment:
Plural wrong?

Instead use:
Types allow your IDE to better assist with code completion during development and only suggest methods and properties for an object that actually exists.